mittab/apps/tab/archive.py

```diff
@@ -0,0 +1,153 @@
+from xml.etree.ElementTree import Element, SubElement, tostring
+
+from .models import School, Team, Judge, Room, Round
+
+DEBATE_ID_PREFIX = "D"
+ROOM_ID_PREFIX = "V"
+SPEAKER_ID_PREFIX = "S"
+TEAM_ID_PREFIX = "T"
+JUDGE_ID_PREFIX = "A"
+SCHOOL_ID_PREFIX = "I"
+SPEAKER_STATUS_PREFIX = "SC"
+
+
+class ArchiveExporter:
+
+    def __init__(self, request, form):
+        self.request = request
```

Comment:
Looks like the request is unused? Maybe we can take that out of this class

Reply:
Actually, more generally, what do you think about refactoring this just to take a `host`, `name` and `start` parameter, and ditch the dependency on the form and request entirely?